
int32 trajectory_id
int32 submap_index
---
bool result

